### PR TITLE
[Docs] Fix Navbar Styling on Mobile Devices

### PR DESCRIPTION
**Pull request by crjm**
This will change static and hovering colors on the navbar when the viewport is small, on both light and dark modes.

Signed-off-by: crjm <julian@dagger.io>

### CODE REVIEW

**Comment by gerhard** [comment]
Leaving this one with you @slumbering 💪

File: website/src/css/custom.scss [hunk 1]
```diff
@@ -615,7 +625,25 @@ li.theme-doc-sidebar-item-link-level-1:active {
...
   html[data-theme="dark"] & {
-    background-color: white;
+    background-color: var(--ifm-color-primary-light);
+
+    kbd.DocSearch-Button-Key {
+      color: var(--ifm-color-primary-light);
+    }
+
+    &:hover .DocSearch-Button-Placeholder,
+    &:active .DocSearch-Button-Placeholder {
```
[the remark this answers is not in the document]
Comment (crjm) [reply website/src/css/custom.scss:635]: Hey! This is for the "Search" word inside the input, it becomes barely visible when hovering in Dark Mode.
image

[the remark this answers is not in the document]
Comment (slumbering) [reply website/src/css/custom.scss:635]: Ok it make sense. ampersand make it hard to read. It even worse when it's referring to an ancestor. I usually use them to refer the direct parent.

```css
.DocSearch-Button {
  html[data-theme="dark"] &:hover .DocSearch-Button-Placeholder,
  html[data-theme="dark"] &:active .DocSearch-Button-Placeholder {
      ...
  }
}
```

More code but easier to understand for a human ;)

**Review by slumbering — APPROVED** [review]
LGTM

**Pull request merged**